### ISSUE TITLE
Duplicate items exist in item database

### ISSUE THREAD

**Issue by osrsbox** [osrsbox/osrsbox-db#9]
A couple of people have reported that there are multiple instances of the same item in the item database. The baseline information for the database contents (item ID, name, cost etc.) comes directly from the OSRS cache. I have deemed this information to be correct, and it is, but it does result in duplicate items. A good example is the Fire cape, which has the following entries:

- Fire cape: 6570 (the actual, equipable version of the cape)
- Fire cape: 10566 (a non equipable version of the cape, cost = 1, the cape also has a different worn model ID number)
- Fire cape: 10637 (another non equipable version of the cape, cost = 60,000)
- Fire cape: 18244 (another non equipable version of the cape, cost = 0, also members is set to false in this item)

### Investigation:

I am _theory crafting_ about the reasons why there are duplicate item IDs that represent the same item. The game must need these multiple IDs for some reason (some logic in the game where an item must be represented multiple times). The following list documents the Fire cape item IDs:

- Fire cape: 6570 (the actual, equipable version of the cape)
- Fire cape: 10566 (the item ID when a Fire cape is dropped)
- Fire cape: 10637 (the item ID when a Fire cape is worn in the wilderness - the increased cost represents the coins dropped on death to the opponent player??)
- Fire cape: 18244 (unknown use! The 0 cost must represent something)

The only way to test this, is to check the item IDs in game. This could be achieved with a custom RuneLite plugin to examine an item and print the item ID to the debug console.

### Solution:

This is a difficult problem to fix, as it is hard to determine the most suitable item ID for a specific item! A potential solution is:

1. Find all the duplicate items (and investigate further, probably manually)
1. For equipable items, always keep the equipable version
1. For tradeable items, always keep the tradeable version
1. Keep the first instance of an item, the lowest number in the cache is (most likely) the original

**Comment by osrsbox** [comment]
This issue should be on the path to being solved. The contents of the database has not been changed, and any duplicate items will be kept. This is because the raw item list is sourced from the game cache, and I have deemed this to be "complete" and "accurate". 

However, I have added additional properties recently to help identify the correct item to use. The useful item attributes/properties are:

- `item.name`
- `item.equipable` or `item.equipable_by_player`
- `item.placeholder`
- `item.noted`
- `item.tradeable` or `item.tradeable_on_ge`

The use of each of these varies on what type of items you want to locate or use.
